instrumentation/spring/spring-cloud-gateway/spring-cloud-gateway-2.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework.cloud")
+    module.set("spring-cloud-starter-gateway")
+    versions.set("[2.0.0.RELEASE,]")
+  }
+}
+
+dependencies {
+  compileOnly("org.springframework.cloud:spring-cloud-starter-gateway:2.0.0.RELEASE")
```

Comment:
Usually we use `library("org.springframework.cloud:spring-cloud-starter-gateway:2.0.0.RELEASE")` for such dependencies. `library` is `compileOnly` + `testImplementation` and allows for testing the latest version similarly to `testLibrary`.